src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Date dateFromDate = new Date();
+        Date dateToDate = new Date();
+        Date date = new Date();
+        StringBuilder builder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo);
+        try {
+            dateFromDate = new SimpleDateFormat("dd.MM.yyyy").parse(dateFrom);
```

Comment:
If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field. https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field